scripts/start-sparkswapd.sh

```diff
@@ -7,7 +7,17 @@
 #            as `/secure` is a volume inside of the container
 ################################################
 
-set -eu
+set -u
+
+# Using host.docker.internal to access the host IP does not work for Linux
+# This is a known issue with Docker: https://github.com/docker/for-linux/issues/264
+# Here, we manually map host.docker.internal to the host IP in /etc/hosts
```

Comment:
One more comment so this description is clearer. We should mention that we map `host.docker.internal` ONLY if the container cannot ping the address (which we assume means we are on linux)